ddtrace/appsec/utils.py

```diff
@@ -1,9 +1,18 @@
+import base64
 import os
+import sys
+from typing import Optional
 
+from ddtrace import constants
 from ddtrace.constants import APPSEC_ENV
+from ddtrace.internal.compat import to_bytes_py2
+from ddtrace.internal.logger import get_logger
 from ddtrace.internal.utils.formats import asbool
 
 
+log = get_logger(__name__)
```

Comment:
We probably shouldn't be introducing a logger here. In general, utilities should `raise` and leave exception handling to the callers of the utility functions. This gives them the flexibility to re-raise to the user in extreme cases or handle the error in whatever way makes sense in context.

Reply:
@brettlangdon suggested we logged a message here.